src/Components/FooterComponent.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import {Footer} from "./Footer.styled";
+
+const FooterComponent = () => {
+  return(
+      <Footer>© &nbsp;
+          {
+              (new Date).getFullYear()
+          }
+      </Footer>
+
```

Comment:
Should it be like this? 

`import * as React from 'react';`
`import {Footer} from "./Footer.styled";`

`const FooterComponent = () => (`
&nbsp;&nbsp;&nbsp;&nbsp;`<Footer> © &nbsp;{new Date().getFullYear()} </Footer>` 
`);`

`export default FooterComponent;`